jobs/rpmbuild.yml

```diff
@@ -57,6 +57,20 @@
     builders:
 #        - system-groovy:
 #            command: build.description = build.getEnvironment().get('fed_repo')
+        - shell: |
+            # Write ci message fields to a file
+            echo "topic=org.centos.prod.ci.pipeline.package.running" >> ${WORKSPACE}/job.properties_running
+            echo "build_url=$BUILD_URL" >> ${WORKSPACE}/job.properties_running
+            echo "build_id=$BUILD_ID" >> ${WORKSPACE}/job.properties_running
+            echo "ref=fedora/${fed_branch}/x86_64/atomic-host" >> ${WORKSPACE}/job.properties_running
+            echo "rev=$fed_rev" >> ${WORKSPACE}/job.properties_running
+            echo "namespace=rpms" >> ${WORKSPACE}/job.properties_running
+            echo "repo=$fed_repo" >> ${WORKSPACE}/job.properties_running
+            echo "status=success" >> ${WORKSPACE}/job.properties_running
+            echo "test_guidance=''" >> ${WORKSPACE}/job.properties_running
```

Comment:
this is fine but I think a cat <<EOF > ${WORKSPACE}/job.properties_running is cleaner.